server/src/test/java/io/spine/server/event/EventBusShould.java

```diff
@@ -355,6 +392,99 @@ public void create_validator_once() {
         assertSame(validator, eventBus.getValidator());
     }
 
+    @Test
+    public void store_an_event() {
+        final Command command = command(createProject());
+        final EBProjectCreatedSubscriber subscriber = new EBProjectCreatedSubscriber();
+        // Register an event subscribe for the event to pass the `DeadEventFilter`
```

Comment:
Please append yourselves as an `@author`. 

Also, please move all of your comments to the Javadoc section of the method (remember to use `{@code ...}` instead of back-ticks).  Several versions ago we decided to move away from the inline comments, as they are hardly readable in comparison to the Javadocs. So let's not increase our technical debt.